lib/internal/http2/compat.js

```diff
@@ -170,12 +225,10 @@ class Http2ServerRequest extends Readable {
     this.on('resume', onRequestResume);
   }
 
-  get closed() {
-    return this[kState].closed;
-  }
-
-  get code() {
-    return this[kState].closedCode;
+  get complete() {
```

Comment:
What's this? I can't see this is a part of the official h1 api?

Reply:
It's not documented but libraries use it.